go/cmd/ocx-sidecar/authcheck.go

```
@@ -0,0 +1,201 @@
+		}
+		decision := gate.Admit(&req)
+		out := authDecision{}
+		if decision.Principal != "" {
+			out.Admitted = true
+			principal := string(decision.Principal)
+			out.Principal = &principal
+		} else {
+			out.Rejection = &authRejection{Status: decision.Rejection.Status, Body: decision.Rejection.Body}
+		}
+		if vector.Probe {
+			sessionState := "missing"
```

golangci-lint (2.13.2)

[error] ineffectual assignment to sessionState

(ineffassign)